game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadLengthReader.java

```diff
@@ -0,0 +1,103 @@
+package games.strategy.engine.framework.map.download;
+// TODO: move to package games.strategy.engine.framework.map.download
```

Comment:
This TODO is redundant as well

Reply:
The intent if not clear is for this TODO to remain until this file is moved, and there are 3 such files to move and each one is marked.